### ISSUE TITLE
Selected hunk in magit becomes grey and white

### ISSUE THREAD

**Issue by joonro** [kuanyui/moe-theme.el#59]
Hi,

This happens for some themes but not for others. With moe-theme, in magit's diff buffer, the selected hunk (where the cursor is)'s highlight changes to grey background and white fonts (Loses green for addition and red for removal), like the following screenshot: 

![image](https://cloud.githubusercontent.com/assets/1063143/7900067/78530bf8-06f7-11e5-946b-844b14d9b6b3.png)

This makes things very hard to read. Some other themes change the background of the selected hunk but keeps the syntax highlighting:

![image](https://cloud.githubusercontent.com/assets/1063143/7900068/b7f29472-06f7-11e5-980f-3f539035c870.png)

It would be great if moe-theme also just highlight the selected hunk with a different background color, but maintains the syntax highlighting.

Thank you!

**Comment by kuanyui** [comment]
The second screenshot you provided seems also not to syntax highlighted.
Or... What you mean is the green&red text for indicate modified parts?

**Comment by joonro** [comment]
Yes - I meant the diff parts. I'm sorry for the confusion!

**Comment by kuanyui** [comment]
I just take a look of definition in `moe-theme`. I think the problem you encountered may be resulted from the conflict with other theme. Try `M-x disable-theme` to disable other theme and see if the problem is still existed.

**Comment by joonro** [comment]
That is strange - I'm sorry I did not catch the screenshot you uploaded (I only saw the email notification). I did `M-x disable-theme` to only have the moe-theme active, but in my machine this happens all the time.

I just tried to turn off everything in my settings and evaluated the following:

```elisp
(require 'moe-theme)
(moe-light)
```

and now it works fine - I will try to pinpoint what is causing the problem.

Thanks!
